CHANGELOG.md

```diff
@@ -45,6 +45,13 @@
 
 # `libp2p` facade crate
 
+# 0.50.1
+
+- Pin pre-releases of alpha dependencies (quic, webrtc & tls) to avoid breaking changes.
+  See [PR XXXX].
+
+[PR XXXX]: https://github.com/libp2p/rust-libp2p/pull/XXXX
```

Comment:
```suggestion
- Pin pre-releases of alpha dependencies (quic, webrtc & tls) to avoid breaking changes.
  See [PR 3540].

[PR 3540]: https://github.com/libp2p/rust-libp2p/pull/3540
```